test/controllers/articles_controller_test.rb

```diff
@@ -35,53 +93,36 @@ def test_create_article_post
 
     # Prove unauthorized user cannot create Article
     login(users(:zero_user).login)
-    post(:create_article, params)
+    post(:create, params)
     assert_flash_text(:permission_denied.l)
     assert_equal(old_count, Article.count)
```

Comment:
```suggestion
    assert_difference("Article.count") do
      post(:create, params)
    end
```
Here and in similar assertions.